letsencrypt/cli.py

```diff
@@ -319,8 +319,7 @@ def _handle_identical_cert_request(config, cert):
     elif config.verb == "certonly":
         keep_opt = "Keep the existing certificate for now"
     choices = [keep_opt,
-               "Renew & replace the cert (limit ~5 per 7 days)",
-               "Cancel this operation and do nothing"]
+               "Renew & replace the cert (limit ~5 per 7 days)"]
 
     display = zope.component.getUtility(interfaces.IDisplay)
     response = display.menu(question, choices, "OK", "Cancel", default=0)
```

Comment:
GitHub won't let me attach a line note to line 326 because it's too far away from your change (!) but the `if` statement just below here is no longer correct after removing this item from the menu. I believe it should change to

```
    if response[0] == "cancel":
```